src/test/java/com/pivovarit/function/ThrowingBiConsumerTest.java

```diff
@@ -11,6 +11,11 @@
 
 class ThrowingBiConsumerTest {
 
+
```

Comment:
To be honest, I have no clue what is it supposed to test. It's definitely not testing anything from the library... will remove it now.

Reply:
https://github.com/pivovarit/throwing-function/commit/650f721de7149bf04d6115a1bf2ecbe778c0e125